arcade/examples/sound_demo.py

```diff
@@ -93,86 +101,30 @@ def setup(self):
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0, volume=volume)
-        button.center_x = SCREEN_WIDTH / 2
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0.5, volume=volume)
-        button.center_x = SCREEN_WIDTH / 4 * 3
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=1, volume=volume)
-        button.center_x = SCREEN_WIDTH - BUTTON_SIZE
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        y = SCREEN_HEIGHT / 2
-        volume = 0.5
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=-1.0, volume=volume)
-        button.center_x = BUTTON_SIZE
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=-0.5, volume=volume)
-        button.center_x = SCREEN_WIDTH / 4
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0, volume=volume)
-        button.center_x = SCREEN_WIDTH / 2
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0.5, volume=volume)
-        button.center_x = SCREEN_WIDTH / 4 * 3
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=1, volume=volume)
-        button.center_x = SCREEN_WIDTH - BUTTON_SIZE
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        y = SCREEN_HEIGHT / 2 - 50
-        volume = 1
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=-1.0, volume=volume)
-        button.center_x = BUTTON_SIZE
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=-0.5, volume=volume)
-        button.center_x = SCREEN_WIDTH / 4
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0, volume=volume)
-        button.center_x = SCREEN_WIDTH / 2
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0.5, volume=volume)
-        button.center_x = SCREEN_WIDTH / 4 * 3
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=1, volume=volume)
-        button.center_x = SCREEN_WIDTH - BUTTON_SIZE
-        button.center_y = y
-        self.button_sprites.append(button)
+        upgrade_sound = ":resources:sounds/upgrade4.wav"
+
+        # fmt: off
+        button_params = [
```

Comment:
Even if we think it looks nice, python style guides strongly disagree with this sort of formatting.

One way to deal with it is to span this across two for loops: the outer one handles setting the row y position, and the inner one passes the `pan` and `vol` columns in.

If you want to be explicit about the panning volumes, maybe you can put those in a tuple constant stored at the top?